src/main/java/com/example/SOPSbackend/security/config/PatientSecurityConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.example.SOPSbackend.security.config;
+
+import com.example.SOPSbackend.model.Patient;
+import com.example.SOPSbackend.security.BasicUserService;
+import com.example.SOPSbackend.security.RestAuthenticationFailureHandler;
+import com.example.SOPSbackend.security.RestAuthenticationSuccessHandler;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(3)
+public class PatientSecurityConfig extends UserSecurityConfig<Patient> {
```

Comment:
Dlaczego to się powtarza?

Reply:
(W sensie wierzę że faktycznie jest jakiś powód)

Reply:
Order(3) pewnie o to chodzi tak? Widzę że chyba i tak się różni tylko jednym słowem w konstruktorze, więc pewnie ciężko będzie wyeliminować.

Reply:
Ok zamykam